roles/tas_single_node/tasks/podman/nginx.yml

```diff
@@ -37,6 +39,14 @@
       | b64decode + (remote_ingress_certificates.results | selectattr('source', 'equalto', tas_single_node_remote_ca) | list | first).content
       | b64decode }}
 
+- name: Base64 encode the rekor ingress
+  ansible.builtin.set_fact:
+    rekor_search_ingress_base64: >-
+      {{ (remote_ingress_certificates.results | selectattr('source', 'equalto', tas_single_node_certs_dir + '/ingress-rekor-search.pem') | list | first).content
+      | b64decode + (remote_ingress_certificates.results | selectattr('source', 'equalto', tas_single_node_remote_ca) | list | first).content
+      | b64decode }}
+
```

Comment:
Another nit: only one line between the tasks, unless there's a reason to create a larger visual space for some reason.

Reply:
I believe this was an issue pointed out by the linitng/sanity checks.